plugin/src/shared/utils/config-utils.ts

```diff
@@ -106,8 +62,45 @@ export const getImportPathPrefix = (tree: Tree): string => {
   return npmScope ? `${npmScope === '@' ? '' : '@'}${npmScope}` : '';
 };
 
-export const verifyEsLintConfig = (tree: Tree): Record<string, any> => {
-  const { config, path } = readESLintConfig(tree);
+export const verifyEsLintConstraintsConfig = (tree: Tree): void => {
+  let constraints: Array<Constraint> = [];
+
+  try {
+    constraints = readConstraints(tree);
+  } catch {
+    output.warn({ title: output.bold('ESLint constraints config not found. Generate default rules...') });
```

Comment:
As the function generates default rules by itself, the second sentence should not be in the imperative mood